website/auth.py

```diff
@@ -26,14 +26,12 @@ def login():
             if user:
                 if check_password_hash(user.password, password):
                     flash('successfully logged in', category='success')
-                return redirect(url_for('views.home')) 
```

Comment:
Should be left as `return redirect(url_for('views.home'))`

Reply:
Resolved in 0a0c6fb